pages/1.8/usage/managing-services/developing-services/overview/index.md

```diff
@@ -139,7 +139,7 @@ Here is a detailed developer workflow for creating a DC/OS service:
 
 # Make your service public
 
-If your service is a Marathon application, you can make it accessible to those outside the cluster by adding labels to your application definition. The DC/OS [Admin Router](https://github.com/mesosphere/adminrouter-public) reads these labels and exposes your app at /service/<marathon-app-name>.
+If your service is a Marathon application, you can make it accessible to those outside the cluster by adding labels to your application definition. The DC/OS [Admin Router](https://docs.mesosphere.com/1.12/overview/architecture/components/#admin-router) reads these labels and exposes your app at /service/<marathon-app-name>.
```

Comment:
this is a link to 1.12 in the 1.8 docs. not sure where this should point to.

Reply:
DC/OS 1.8 architecture docs does not call out `Admin Router` info in detail. So it is more appropriate to point to https://docs.mesosphere.com/1.8/overview/components/ instead of 1.12 docs link.